MessageStackView/MessageManager.swift

```diff
@@ -9,12 +9,48 @@
 import Foundation
 import UIKit
 
-/// Controller for posting aInd removing `Message`s on a `UIStackView`.
+// MARK: - MessageManagerDelegate
+
+/// Delegate methods for `MessageManager`
+public protocol MessageManagerDelegate: AnyObject {
+    
+    /// Called when a `view` will be posted
+    func messageManager(_ messageManager: MessageManager, willPost view: UIView)
```

Comment:
Try to use the Xcode shortcut when documenting functions (cmd + alt + /) as it will auto-fill documentation bits for your function variables too :D